src/status_im/wallet/prices.cljs

```diff
@@ -33,9 +33,13 @@
 (fx/defn on-update-prices-success
   {:events [::update-prices-success]}
   [{:keys [db]} prices]
-  {:db (assoc db
-              :prices prices
-              :prices-loading? false)})
+  {:db (if (contains? prices :USD)
```

Comment:
but what if a user uses different currency? probably we need more general solution and check not USD but user currency ?

Reply:
yes, it's more generic.